compiler/test/fail_compilation/issue24246.d

```diff
@@ -0,0 +1,27 @@
+// https://issues.dlang.org/show_bug.cgi?id=24246
+
+/*
+TEST_OUTPUT:
+---
+fail_compilation/issue24246.d(116): Error: CTFE internal error: literal `int` could not be copied.
+---
+*/
+
+#line 100
+
+auto f24246()
+{
+    return 1;
+}
+
+auto f24246(int i)
+{
+    return true ? int : i;
```

Comment:
When do such a primary as `int` is accepted since ?

It is like https://issues.dlang.org/show_bug.cgi?id=24229.

Reply:
That comes from an alias presumably ? IdentExp -> AliasDEcl -> TypeExp ?